app/views/layouts/application.html.erb

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
-    <%= csrf_meta_tags %>
+  <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
```

Comment:
` Ruby on Rails Tutorial Sample App`
em tìm hiểu về I18n trong rails, và sử dụng `I18n ` cho TOÀN BỘ TEXT của toàn bộ các file html nhé

Reply:
Comment này chưa fix em ơi